packages/observable/src/mappingHelpers.ts

```diff
@@ -55,12 +59,14 @@ function mapJsObjectGraph (rootObject, mapInputCallback, visitedObjects = new Ma
   return outputProperties
 }
 
-function visitPropertiesOrArrayEntries (rootObject, visitorCallback) {
+function visitPropertiesOrArrayEntries<T = any>(rootObject: T, visitorCallback: (indexer: any) => void): void {
   if (rootObject instanceof Array) {
-    for (var i = 0; i < rootObject.length; i++) { visitorCallback(i) }
+    for (var i = 0; i < rootObject.length; i++) {
+      visitorCallback(i)
+    }
 
         // For arrays, also respect toJSON property for custom mappings (fixes #278)
-    if (typeof rootObject['toJSON'] === 'function') { visitorCallback('toJSON') }
+  if (typeof rootObject['toJSON'] === 'function') { visitorCallback('toJSON') }
```

Comment:
wrong indent?

Reply:
Fixed in https://github.com/Auge19/tko/pull/4